examples/cpp-simple/server.cc

```diff
@@ -16,101 +16,107 @@
 // This example really does nothing other than show how to integrate
 // the C++ SDK.
 
-#include <iostream>
-#include <thread>
 #include <agones/sdk.h>
 #include <grpc++/grpc++.h>
+#include <iostream>
+#include <thread>
 
 // send health check pings
-void doHealth(agones::SDK *sdk) {
-    while (true) {
-        if (!sdk->Health()) {
-            std::cout << "Health ping failed" << std::endl;
-        } else {
-            std::cout << "Health ping sent" << std::endl;
-        }
-        std::this_thread::sleep_for(std::chrono::seconds(2));
+void DoHealth(std::shared_ptr<agones::SDK> sdk) {
+  while (true) {
+    if (!sdk->Health()) {
+      std::cout << "Health ping failed\n";
+    } else {
+      std::cout << "Health ping sent\n";
     }
+    std::this_thread::sleep_for(std::chrono::seconds(2));
+  }
 }
 
 // watch GameServer Updates
-void watchUpdates(agones::SDK *sdk) {
-    std::cout << "Starting to watch GameServer updates..." << std::endl;
-    sdk->WatchGameServer([](stable::agones::dev::sdk::GameServer gameserver){
-        std::cout << "GameServer Update, name: " << gameserver.object_meta().name() << std::endl;
-        std::cout << "GameServer Update, state: " << gameserver.status().state() << std::endl;
-    });
+void WatchUpdates(std::shared_ptr<agones::SDK> sdk) {
+  std::cout << "Starting to watch GameServer updates...\n";
+  sdk->WatchGameServer([](agones::dev::sdk::GameServer gameserver) {
+    std::cout << "GameServer Update, name: " << gameserver.object_meta().name()
```

Comment:
(optional) clang-format tip: You an influence the way clang-format wraps lines by using an end-of-line comment. This can be useful when you want to group semantically meaning things on a single line, whereas clang-format will only care about syntax, not semantics. For example, you might consider formatting this output like:

```cc
std::cout << "GameServer update:\n"                                 //
          << "\tname: " << gameserver.object_meta().name() << "\n"  // 
          << "\tstate: " << gameserver.status().state() << "\n";
```

Reply:
Neat trick. Done.